src/Util/ProxyUtils.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace Friendica\Util;
+
+use Friendica\BaseModule;
+use Friendica\BaseObject;
+use Friendica\Core\Config;
+use Friendica\Core\System;
+
+require_once 'include/security.php';
+
+/**
+ * @brief Proxy utilities
+ */
+class ProxyUtils
+{
+
+	/**
+	 * Default time to keep images in proxy storage
+	 */
+	const DEFAULT_TIME = 86400; // 1 Day
+
+	/**
+	 * Sizes constants
+	 */
+	const SIZE_MICRO  = 'micro';
+	const SIZE_THUMB  = 'thumb';
+	const SIZE_SMALL  = 'small';
+	const SIZE_MEDIUM = 'medium';
+	const SIZE_LARGE  = 'large';
+
+	/**
+	 * Accepted extensions
+	 *
+	 * @var array
+	 * @todo Make this configurable?
+	 */
+	private static $extensions = [
```

Comment:
Can't we use `ProxyUtils` value for this?

Reply:
Thank you finding these. Must be a left-over from splitting classes.